src/pages/project/project_wizard/RelatedProjectsField.tsx

```diff
@@ -62,15 +69,28 @@ export const RelatedProjectsField = () => {
           />
         )}
       </Field>
-      <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.25rem', paddingLeft: 0 }} component={'ul'}>
-        {values.relatedProjects.map((relatedProject) => (
-          <RelatedProjectItem
-            projectId={relatedProject}
-            key={relatedProject}
-            removeProject={() => removeProject(relatedProject)}
-          />
-        ))}
-      </Box>
+      {values.relatedProjects.length > 0 && (
```

Comment:
I `RelatedRegistrationsField` er det også lagt til en tekst for ingen treff når det ikke er noen. Skulle det også vært her, eller er det utelatt bevisst?

Reply:
Nei det er sant, vi bør nok ha det der også nå siden det er paginering nå.